src/main/java/me/gnat008/perworldinventory/config/DatabaseProperties.java

```diff
@@ -0,0 +1,68 @@
+package me.gnat008.perworldinventory.config;
+
+import com.github.authme.configme.Comment;
+import com.github.authme.configme.SectionComments;
+import com.github.authme.configme.SettingsHolder;
+import com.github.authme.configme.properties.Property;
+import me.gnat008.perworldinventory.data.DataSourceType;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.github.authme.configme.properties.PropertyInitializer.newProperty;
+
+/**
+ * Holds the properties for the SQL database
+ */
+public final class DatabaseProperties implements SettingsHolder {
+
+    @Comment({"Type of database to use",
+        "Valid values: flatfile, sqlite, mysql"})
```

Comment:
True. I'm in a weird place of figuring it will be done with the MySQL release, but I just don't have everything for it yet.